src/st/compatibility/class-wpml-page-builders-update.php

```diff
@@ -47,6 +47,16 @@ private function save_data( $post_id, $fields, $data ) {
 	 */
 	private function copy_meta_fields( $translated_post_id, $original_post_id, $meta_fields ) {
 		foreach ( $meta_fields as $meta_key ) {
+			if ( 'post_content' === $meta_key ) {
+				$original_post = get_post( $original_post_id );
+				wp_update_post( array(
+					'ID' => $translated_post_id,
+					'post_content' => $original_post->post_content,
+				) );
+
+				continue;
```

Comment:
@adrianosferreira I would remove the `continue` and put the copy of the meta values in the `else`

Reply:
@brucepearson done.